src/DDTrace/Integrations/Curl/CurlIntegration.php

```diff
@@ -44,12 +44,17 @@ public static function load()
 
         dd_trace('curl_exec', function ($ch) use ($integration) {
             $tracer = GlobalTracer::get();
+            if ($tracer->limited()) {
+                CurlIntegration::injectDistributedTracingHeaders($ch);
```

Comment:
What do you think of moving `CurlIntegration::injectDistributedTracingHeaders($ch);` as the first line so we can avoid repetition between this call and the one on line 58?

Reply:
I considered that, and looking at implementation of:
```
   CurlIntegration::injectDistributedTracingHeaders($ch);
```

It takes current active span. Which means this needs to be called in two different places.

Reply:
Good point! 👍 resolving then